src/pip/_internal/commands/install.py

```diff
@@ -249,6 +250,7 @@ def run(self, options, args):
 
         install_options = options.install_options or []
 
+        logger.debug("Using {}".format(get_pip_version()))
```

Comment:
I think a test can be added to verify that the logger contains this string when install in called with `-v`, something along the lines of [tests.lib.test_lib:https://github.com/pypa/pip/blob/master/tests/lib/test_lib.py#L40](https://github.com/pypa/pip/blob/master/tests/lib/test_lib.py#L40), maybe a smaller version of it.